support-frontend/assets/pages/supporter-plus-thank-you/supporterPlusThankYou.tsx

```diff
@@ -140,14 +141,32 @@ export function SupporterPlusThankYou(): JSX.Element {
 	const isAmountLargeDonation = amount
 		? isLargeDonation(amount, contributionType, paymentMethod)
 		: false;
-
-	const promotion = getPromotion(
-		productPrices,
-		countryId,
-		billingPeriod,
-		fulfilmentOption,
-		productOption,
+	const thresholdPrice = useContributionsSelector((state) =>
+		getThresholdPrice(contributionType, state),
 	);
+	/**
+	 * We would normally use the isSuporterPlusFromState selector here,
+	 * but the amount can actually come from `localStorage`.
+	 *
+	 * We should clear this up when refactoring
+	 */
+	const isSupporterPlus = thresholdPrice ? amount >= thresholdPrice : false;
```

Comment:
Do we want to check `contributionType !== 'ONE_OFF'` here too? EG. I'm wondering if a one off contribution with an amount above the threshold will lead to `isSupporterPlus`  being set as `true`... Unless `thresholdPrice` is a `falsy` value if `contributionType` is 'ONE_OFF'? In which case we don't need to check `contributionType !== 'ONE_OFF'` here.

Reply:
Done - hopefully we'll be able to make this more specific to product soon.